src/coreclr/jit/simdashwintrinsic.cpp

```diff
@@ -753,6 +798,48 @@ GenTree* Compiler::impSimdAsHWIntrinsicSpecial(NamedIntrinsic       intrinsic,
                     return gtNewSimdAbsNode(retType, op1, simdBaseJitType, simdSize, /* isSimdAsHWIntrinsic */ true);
                 }
 
+                case NI_VectorT128_ConvertToDouble:
+                {
+                    assert((simdBaseType == TYP_LONG) || (simdBaseType == TYP_ULONG));
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_Arm64_ConvertToDouble, simdBaseJitType,
+                                                    simdSize, /* isSimdAsHWIntrinsic */ true);
+                }
+
+                case NI_VectorT128_ConvertToInt32:
+                {
+                    assert(simdBaseType == TYP_FLOAT);
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_ConvertToInt32RoundToZero, simdBaseJitType,
+                                                    simdSize, /* isSimdAsHWIntrinsic */ true);
+                }
+
+                case NI_VectorT128_ConvertToInt64:
+                {
+                    assert(simdBaseType == TYP_DOUBLE);
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_Arm64_ConvertToInt64RoundToZero,
+                                                    simdBaseJitType, simdSize, /* isSimdAsHWIntrinsic */ true);
+                }
+
+                case NI_VectorT128_ConvertToSingle:
+                {
+                    assert((simdBaseType == TYP_INT) || (simdBaseType == TYP_UINT));
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_ConvertToSingle, simdBaseJitType, simdSize,
+                                                    /* isSimdAsHWIntrinsic */ true);
+                }
+
+                case NI_VectorT128_ConvertToUInt32:
+                {
+                    assert(simdBaseType == TYP_FLOAT);
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_ConvertToUInt32RoundToZero,
+                                                    simdBaseJitType, simdSize, /* isSimdAsHWIntrinsic */ true);
+                }
+
+                case NI_VectorT128_ConvertToUInt64:
+                {
+                    assert(simdBaseType == TYP_DOUBLE);
+                    return gtNewSimdHWIntrinsicNode(retType, op1, NI_AdvSimd_Arm64_ConvertToUInt64RoundToZero,
+                                                    simdBaseJitType, simdSize, /* isSimdAsHWIntrinsic */ true);
+                }
```

Comment:
It's worth calling out these weren't accelerated at all on ARM64 before; and now they are and via a single instruction.